instrumentation/net/http/otelhttp/internal/semconv/bench_test.go

```diff
@@ -43,6 +39,6 @@ func BenchmarkHTTPServerRequest(b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
 	for range b.N {
-		benchHTTPServerRequestResults = serv.RequestTraceAttrs("", req, RequestTraceAttrsOpts{})
+		_ = serv.RequestTraceAttrs("", req, RequestTraceAttrsOpts{})
```

Comment:
Does the compiler now optimize away this function call because the result is never used?

Having the result stored outside of the call scope ensures this optimization is not made. It is not clear that is still the case.